charts/sequencer/templates/configmaps.yaml

```diff
@@ -43,35 +43,35 @@ data:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Values.config.moniker }}-cometbft-env
+  name: {{ .Values.moniker }}-cometbft-env
   namespace: {{ include "sequencer.namespace" . }}
 data:
-  COMET_BFT_RPC_PORT: "{{ .Values.ports.cometBFTRPC }}"
+  COMET_BFT_RPC_PORT: "{{ .Values.ports.cometbftRpc }}"
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Values.config.moniker }}-sequencer-env
+  name: {{ .Values.moniker }}-sequencer-env
   namespace: {{ include "sequencer.namespace" . }}
 data:
   ASTRIA_SEQUENCER_LOG: "astria_sequencer=debug"
   ASTRIA_SEQUENCER_LISTEN_ADDR: "127.0.0.1:{{ .Values.ports.sequencerABCI }}"
   ASTRIA_SEQUENCER_DB_FILEPATH: "/sequencer/penumbra.db"
   ASTRIA_SEQUENCER_ENABLE_MINT: "false"
-  # Socket address for gRPC server
-  ASTRIA_SEQUENCER_GRPC_ADDR: "0.0.0.0:{{ .Values.ports.sequencerGRPC }}"
-  ASTRIA_SEQUENCER_NO_METRICS: "{{ not .Values.config.sequencer.metrics.enabled }}"
+  # Socket address for GRPC server
+  ASTRIA_SEQUENCER_GRPC_ADDR: "0.0.0.0:{{ .Values.ports.sequencerGRpc }}"
```

Comment:
nit: I think `sequencerGRPC` is technically correct here as the protocol is officially `gRPC`.

I also know that we do `Rpc` instead of `RPC` given that I think I prefer `sequencerGrpc` over `serquencerGRpc` which just feels off somehow lol.